src/Stack/Stack.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { ReactElement } from "react"
+import styled from "@emotion/styled"
+import { TAlign } from "@new/Align/Align"
+import { containsIlligalChildren } from "@new/Functions"
+import { computeColor, EColor, TColor } from "@new/Color"
+import { TLayoutBase } from "@new/Composition/TLayoutBase"
+import { Loader } from "./internal/Loader"
+import { Spinner } from "./internal/Spinner"
+
+const translateBorderRadius = (p?: Pick<TStack, "borderRadius">): string => {
```

Comment:
Why not do this to avoid always wrapping the arg in a object when calling this function?
```suggestion
const translateBorderRadius = (p?: TStack["borderRadius"]): string => {
```

Reply:
Because Pick is the "TypeScript way" of doing this. It's more visible that you are picking specific properties rather than string-indexing.